time    stamp
quad_control/UAVPose position
quad_control/UAVPose twist
quad_control/UAVPose wrench